works.html

```diff
@@ -0,0 +1,103 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="work-styles.css">
+</head>
+<body>
+
+  <header>
+    <h1>My Recent Works</h1>
+  </header>
```

Comment:
- [ ] After you put this section in the `index.html` file kindly make sure you grab this title in a section tag as well since it is the title of the section.